client/src/components/GroupsList/GroupsList.css

```diff
@@ -0,0 +1,48 @@
+
+.Groups {
+  width: 70vw;
+}
+.Groups__list {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+}
+
+.Groups__card--1, .Groups__card--2, .Groups__card--3 {
+  margin: 1em 1em 0 0;
+} 
+
+.Groups__card--4 {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 125px;
+  height: 150px;
+  text-align: center;
+}
+
+.Groups h1 {
```

Comment:
@tissakhosla Will you give this a className that is semantic and follows the BEM?

Reply:
NP... done :)